.github/workflows/publish.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  pypi:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    environment:
+      name: pypi
+      url: https://pypi.org/p/cedar-solve
```

Comment:
@smroid is this the correct pypi project name?

Reply:
Yes.